contrib/win32/win32compat/wmain_sshd.c

```diff
@@ -224,14 +224,40 @@ int argc_original = 0;
 wchar_t **wargv_original = NULL;
 
 int wmain(int argc, wchar_t **wargv) {
-	wchar_t* path_utf16;
+	wchar_t *path_value = NULL, *path_new_value;
+	errno_t result = 0;
+	size_t path_new_len = 0, len;
 	argc_original = argc;
 	wargv_original = wargv;
 
 	init_prog_paths();
 	/* change current directory to sshd.exe root */
 	_wchdir(__wprogdir);
 
+	_wdupenv_s(&path_value, &len, L"PATH");
```

Comment:
Please add comments on why we are doing this

Reply:
added.